style/index.css

```diff
@@ -122,20 +122,33 @@ div.CodeMirror-linebackground.jp-InputArea-editor-dependencyline {
 }
 
 .jp-OutputArea-highlighted .jp-OutputArea-gatherbutton {
-  border-radius: 3px;
-  border: 1px solid black;
+  border: 1px solid var(--brand-color3);
+  border-radius: 3px 3px 0px 0px;
+  border-bottom-style: none;
   cursor: pointer;
-  padding: 0.5em 1em;
+  padding: 0.25em 1em;
   position: absolute;
-  top: 10px;
+  height: min-content;
   right: 10px;
   background: white;
   font-family: var(--jp-ui-font-family);
-  opacity: 0.9;
+  font-size: 12px;
+  line-height: 19px;
```

Comment:
Make font size and line height follow from `--jp-` variables (`--jp-ui-font-size1`, `--jp-content-line-height`), see available variables at https://github.com/jupyterlab/jupyterlab/blob/c3d6801290fd5cdb527d1a9a198f97c19f890a1f/packages/theme-light-extension/style/variables.css#L106-L110

Reply:
I specified these because the button sizes are different if line height isn't defined for the button style (i.e. df vs. text).

Reply:
Yup that makes sense. Will it still work if you define font size and line height like:

```css
font-height: var(--jp-ui-font-size1);
line-size: var(--jp-content-line-height);
```

To my knowledge the values of these variables are very similar to your literals, and would get updated to match the rest of the UI if the font defaults changed.